csrc/id_model/id_model.h

```diff
@@ -281,4 +309,11 @@ class IdModel : public PolymorphicBase {
   std::unordered_map<ValGroup, IterDomain*> loop_promotion_map_;
 };
 
+// A utility function to update a map of ValGroups to ID from an old
+// Valgraph to a new ValGraph. The new graph must be a superset of the
+// old graph.
+std::unordered_map<ValGroup, IterDomain*> updateValGroupIdMap(
```

Comment:
Is this only used in unit tests?

Reply:
ATM, yes. It will be used in the next step as well.